### ISSUE TITLE
Signal links / Wireless wires

### ISSUE THREAD

**Issue by UniDuki** [studio-minus/ppg-bugs-and-requests#199]
**Feature description**
You drag it between two objects and then they get linked, it will have a line between the two links, only visible in detail view, just like a phase link. Once two objects have been linked, they will behave almost like radios, where sending a signal to one will send the same signal to the other linked object. 

**Feature Purpose**
Yes we have wireless signals in the form of radios. But lets say you have a contraptions that needs radios to function, that's fine. But now you try spawn four of the contraption, and now they all break because they are interfering with each other.

**Comment by TamasB14** [comment]
I'd like Signal links too, working similiar to wires, but that'd make radio pretty useless unless RP purposes. Still... can't wait to build a Bluetooth system ingame.

**Comment by LaCumbiaDelCoronavirus** [comment]
Im pretty sure, you can make a near-perfect but not "compact" system of this just by 'transmitting' with a radio and with the other radio also having this activation-electricity-transformer type machine, forgot the name

**Comment by StealthDrifter** [comment]
you can change the channel of the radio??
you just described exactly how the radio works but instead of channels its a wire

**Comment by UniDuki** [comment]
> you can change the channel of the radio??
> you just described exactly how the radio works but instead of channels its a wire

No, that is not exactly how the radio works. Radios work by sending the signal to **all** other radios on the same channel. What I'm asking for is a wireless link between just 2 objects.
If you spawn 10 of the exact same contraption that uses radios, do you really want to painstakingly go through each radio in each contraption and make them slightly different so that they wont mess with each other, or make 10 versions of the same contraption just with slightly different radio channels? Or would you rather do what I want, simply use a signal link instead of radios so that they will no longer interfere with each other but still work just as well.
